site-src/guides/metrics.md

```diff
@@ -86,11 +105,13 @@ metadata:
     kubernetes.io/service-account.name: inference-gateway-sa-metrics-reader
 type: kubernetes.io/service-account-token
 ```
-Then, you can curl the 9090 port like following
+
+Then, you can curl the appropriate port as follows. For EPP (port 9090),
```

Comment:
nit: remove the `,` at the end of the sentence.

Reply:
I updated it in [7269f88](https://github.com/kubernetes-sigs/gateway-api-inference-extension/pull/980/commits/7269f882d7f14d360dedb30e509ecae71628bd94), thanks!